operations/configuration_management/CFS_Commands_Cheat_Sheet.md

```diff
@@ -36,6 +36,9 @@ For more information about the CFS API, see [Configuration Framework Service](..
 
 * (`ncn-mw#`) Clear the state and trigger configuration on a component:
 
+    > For any of these components with a desired configuration set, this will trigger automatic configuration.
```

Comment:
```suggestion
* (`ncn-mw#`) Clear the state of a component:

    > If this component has a desired configuration set, this will trigger automatic configuration.
```